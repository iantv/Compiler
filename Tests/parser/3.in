int a;

void main(int a){
	int b;
	int c;
}