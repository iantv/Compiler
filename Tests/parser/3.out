main: function
	a: parameter
		int
	{
		b: variable
			int
		c: variable
			int
	}
returns
	void
a: variable
	int
